django_icons/renderers/image.py

```diff
@@ -0,0 +1,73 @@
+import os
+
+from django.conf import settings
+from django.forms.utils import flatatt
+from django.utils.html import format_html
+from django.utils.safestring import mark_safe
+
+from django_icons.css import merge_css_text
+from django_icons.renderers.base import BaseRenderer
+
+
+class ImageRenderer(BaseRenderer):
+    """
+    Render an icon from a local file
```

Comment:
Maybe explain the general working of the renderer here. It looks like this is built for one image per icon. I;d mention that, since there are also image icon systems that have one big image which is partially shown based on icon CSS classes.